examples/version-migration/same-container/tests/sameContainer.test.ts

```diff
@@ -149,11 +151,13 @@ describe("same-container migration", () => {
 
 			// Get a promise that will resolve when both sides have finished migration
 			const migrationP = page.evaluate(() => {
-				const migrationPs = (window["migrators"] as IMigrator[]).map((migrator) => {
-					return new Promise<void>((resolve) => {
-						migrator.once("migrated", resolve);
-					});
-				});
+				const migrationPs = (window["migrators"] as ISameContainerMigrator[]).map(
+					(migrator) => {
+						return new Promise<void>((resolve) => {
+							migrator.once("migrated", resolve);
```

Comment:
Can you clarify why it's not `migrator.events.once` here?

Reply:
In this PR I'm only changing the separate container flow, so ISameContainerMigrator still uses inheritance.